static/app/views/insights/cache/settings.ts

```diff
@@ -3,6 +3,8 @@ import type {SpanMetricsQueryFilters} from 'sentry/views/insights/types';
 
 export const MODULE_TITLE = t('Caches');
 export const BASE_URL = 'caches';
+export const DATA_TYPE = t('Cache');
+export const DATA_TYPE_PLURAL = MODULE_TITLE;
```

Comment:
```suggestion
export const DATA_TYPE_PLURAL = t('Caches');
```

Safer, so they can be changed independently. If you change the `MODULE_TITLE` to `t('Cache Insights')` or something, it'd look weird in other places

Reply:
Yeah that's a good point, I'll change it! 

I was trying to decide between reusing module title or not, but your of point of it being safer to not reuse it makes sense